lib/lev/routine.rb

```diff
@@ -206,34 +214,6 @@ def [](*args, &block)
         result.outputs.send(@express_output)
       end
 
-      if defined?(ActiveJob)
-        def active_job_class
-          @active_job_class ||= const_set("ActiveJob",
-            Class.new(Lev.configuration.active_job_class) do
-              queue_as do
-                parent_routine.active_job_queue
-              end
-
-              def parent_routine
-                self.class.parent
-              end
-
-              def perform(*args, &block)
-                parent_routine.call(*args, &block)
-              end
-            end
-          )
-        end
-
-        def perform_later(*args, &block)
-          active_job_class.perform_later(*args, &block)
-        end
-
-        def active_job_queue
```

Comment:
Can you leave this in?  (actually rename it `background_queue` and change the appropriate code in https://github.com/lml/lev/blob/560d28f486f0c0fd67a17637a3fb2e61193972bf/lib/lev/object.rb#L11).

Then below in `enable_background_jobs` you can use this variable which can be set per routine.